frontend/src/styles.scss

```diff
@@ -54,5 +54,12 @@ body {
     color: #ffffff;
     background-color: #7e7777;
   }
-  
+  input#item-name, input#item-price {
```

Comment:
This is actually dark-mode controlled and currently dark-mode logic is applied in the styles.scss.  The plan is to eventually get this all refactored down the road.  For now, it's just a matter of getting things to work, especially since I'm still learning Angular.  You are more than welcome to make such adjustments at any time though  Thx!

Reply:
All clear! :)